nautilus_trader/data/aggregation.pxd

```diff
@@ -53,7 +53,7 @@ cdef class BarBuilder:
 
     cpdef void set_partial(self, Bar partial_bar)
     cpdef void update(self, Price price, Quantity size, uint64_t ts_event)
-    cpdef void update_bar(self, Bar bar)
+    cpdef void update_bar(self, Bar bar, Quantity size, uint64_t ts_init)
```

Comment:
Do we need to separate `size` from `bar` and then have to do `update_bar(bar, bar.volume)` ?

Reply:
update from size to volume variable name done